packages/grafana-ui/src/components/Combobox/useComboboxFloat.ts

```diff
@@ -23,7 +25,7 @@ export const useComboboxFloat = (
 ) => {
   const inputRef = useRef<HTMLInputElement>(null);
   const floatingRef = useRef<HTMLDivElement>(null);
-  const [popoverMaxWidth, setPopoverMaxWidth] = useState<number | undefined>(undefined);
+  const [popoverMaxSize, setPopoverMaxSize] = useState<{ width: number; height: number } | undefined>(undefined);
```

Comment:
Because they're set and used together, to me they feel more suited to be one object, but I don't have a strong preference about it.

I think I'll just leave it as-is unless you feel stronger about it than me 😅